### PR TITLE
Jungle dragon

### PR DESCRIPTION
**Pull request by Moltijoe**
ty to crazy drakkon for the sprites

Is effectively the same as the ash drake, but with a few minor tweaks
-lower melee damage
-shorter ability cooldown
-tries to maintain distance
-when it does the nova attack, it also has a chance to do meteors
-minor tweak to attack probabilities
-damage types changed to better fit the new visuals

# Why is this good for the game?
jungleland unique megafauna that is still one that people know

# Testing
image

# Spriting
dragon

:cl:  crazy_drakkon & molti
rscadd: Adds Jungle Dragon megafauna
imageadd: adds sprites to jungle dragon
/:cl:

### CODE REVIEW

File: yogstation/code/modules/jungleland/megafauna/jungle_drake.dm [hunk 1]
```diff
@@ -0,0 +1,376 @@
+/mob/living/simple_animal/hostile/megafauna/dragon/jungle
+	name = "jungle dragon"
+	desc = "Warden of the viridian."
```
Comment (AMyriadseviltwin) [inline yogstation/code/modules/jungleland/megafauna/jungle_drake.dm]:
```suggestion
	desc = "Guardians of the jungle, they seem to wander the lands in search of something."
```

File: yogstation/code/modules/jungleland/megafauna/jungle_drake.dm [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+	icon = 'yogstation/icons/mob/jungle64x64.dmi'
+	gps_name = "Verdant Signal"
```
Comment (AMyriadseviltwin) [inline yogstation/code/modules/jungleland/megafauna/jungle_drake.dm]: Two more options, feel free to choose any of them or keep what's there.

```suggestion
	gps_name = "Mossy Signal"
```
or

```suggestion
	gps_name = "Viridian Signal"
```

File: yogstation/code/modules/jungleland/megafauna/jungle_drake.dm [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+	deathmessage = "collapses to the floor, its strength long since exhausted."
+
```
Comment (AMyriadseviltwin) [inline yogstation/code/modules/jungleland/megafauna/jungle_drake.dm]:
```suggestion
	weather_immunities = WEATHER_ACID
	
```

File: yogstation/code/modules/jungleland/megafauna/jungle_drake.dm [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+			// deals damage to mechs
+			for(var/obj/mecha/M in T.contents)
+				if(M in hit_list)
+					continue
+				hit_list += M
+				M.take_damage(45, BRUTE, MELEE, 1)
+		sleep(0.15 SECONDS)
+
+/obj/effect/temp_visual/vineball
+	name = "Vine"
```
Comment (AMyriadseviltwin) [inline yogstation/code/modules/jungleland/megafauna/jungle_drake.dm]:
```suggestion
	name = "barbed vines"
```

File: yogstation/code/modules/jungleland/megafauna/jungle_drake.dm [hunk 5]
```diff
@@ -0,0 +1,376 @@
...
+	desc = "A jungle drakes vine something or other, idk."
```
Comment (AMyriadseviltwin) [inline yogstation/code/modules/jungleland/megafauna/jungle_drake.dm]:
```suggestion
	desc = "Not the kind you want to brush up against, move!"
```

File: yogstation/code/modules/jungleland/megafauna/jungle_drake.dm [hunk 6]
```diff
@@ -0,0 +1,376 @@
...
+	// changes turf to toxic water temporarily
+	if(!istype(T, /turf/closed) && !istype(T, /turf/open/water/toxic_pit/deep))
+		var/lava_turf = /turf/open/water/toxic_pit/deep
+		var/reset_turf = T.type
+		T.ChangeTurf(lava_turf, flags = CHANGETURF_INHERIT_AIR)
+		sleep(reset_time)
+		T.ChangeTurf(reset_turf, flags = CHANGETURF_INHERIT_AIR)
+
+/obj/effect/temp_visual/drakewall/jungle
+	name = "Vine Barrier"
```
Comment (AMyriadseviltwin) [inline yogstation/code/modules/jungleland/megafauna/jungle_drake.dm]:
```suggestion
	name = "dense vines"
```

File: yogstation/code/modules/jungleland/megafauna/jungle_drake.dm [hunk 7]
```diff
@@ -0,0 +1,376 @@
...
+	desc = "A jungle drakes vine something or other, idk."
```
Comment (AMyriadseviltwin) [inline yogstation/code/modules/jungleland/megafauna/jungle_drake.dm]:
```suggestion
	desc = "So thick you can barely move through it!"
```

**Review by AMyriadseviltwin — COMMENTED** [review]
lore/grammar/writing pass 👍

**Comment by Mantlecrawler** [comment]
you should definitely make this its fight music
https://www.youtube.com/watch?v=8rEw9sCwo8E

**Comment by neeshacark** [comment]
it might just be because im comparing it to the fantastic ash drake sprite, but this dragon seems a little underdetailed to me

**Comment by Moltijoe** [comment]
> it might just be because im comparing it to the fantastic ash drake sprite, but this dragon seems a little underdetailed to me

🤷‍♂️ i think it looks fine, but yeah, with how much stuff the ash drake has on it, i can understand that sentiment

**Comment by cuackles** [comment]
> it might just be because im comparing it to the fantastic ash drake sprite, but this dragon seems a little underdetailed to me

It's because they are on diferent styles, ash drake has a lot more color shades, it has a lot more fuzzy details but if you zoom in it's harder to tell what's going on, this one you can clearly see what's what, no saying one style is superior to the other but that's the reason why this one could look slightly more empty

**Review by cuackles — DISMISSED** [review]

**Comment by CrazyDrakkon** [comment]
> it might just be because im comparing it to the fantastic ash drake sprite, but this dragon seems a little underdetailed to me

I'd consider myself a beginner but I had trouble deciding how to texture it. I raise you pickl- i mean scales.
Jungle Drake +scales

Edit: Final
jungle_drake

**Comment by Cartlord** [comment]
...what does it do, though? is it just an ash drake reskin?

File: _maps/RandomRuins/JungleRuins/jungleland_jungle_dragon.dmm [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+/obj/structure/flora/junglebush/large,
+/turf/open/floor/plating/dirt/jungleland/jungle,
+/area/template_noop)
+"r" = (
+/obj/structure/flora/junglebush/c,
+/turf/open/floor/plating/dirt/jungleland/jungle,
+/area/template_noop)
+"E" = (
+/obj/structure/flora/rock/pile/largejungle,
+/mob/living/simple_animal/hostile/megafauna/dragon/jungle/,
```
Comment (AMyriad) [inline _maps/RandomRuins/JungleRuins/jungleland_jungle_dragon.dmm]:
```suggestion
/mob/living/simple_animal/hostile/megafauna/dragon/jungle,
```
lol mb

**Comment by AMyriad** [comment]
need someone else to review maps as I took part in map changes here

**Review by Aquizit — APPROVED** [review]

**Review by cuackles — APPROVED** [review]

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**